plugins/CoreAdminHome/OptOutManager.php

```diff
@@ -145,6 +199,16 @@ public function getOptOutView()
             ? $language
             : LanguagesManager::getLanguageCodeForCurrentUser();
 
+        $this->addQueryParameters(array(
+            'module' => 'CoreAdminHome',
+            'action' => 'optOut',
+            'language' => $lang,
+        ), false);
+
+        if ($this->isUserAgentSafari() && $trackVisits) {
```

Comment:
Btw: there is PR with changes to always set the cookie in a new window: https://github.com/piwik/piwik/pull/8646 - this will affect / simplify this PR